domain-scan/scripts/nmap-basic.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-SCAN_DIRECTORY="${1}"
+OUTPUT_DIRECTORY="${1}"
```

Comment:
Thanks for changing this. We should be consistent about passing in directory versus using the exported definitions. I'll probably create an issue for that once we have time to do some cleanup.